src/game_classification.hpp

```diff
@@ -72,21 +66,21 @@ class game_classification
 
 	bool is_scenario() const
 	{
-		return campaign_type == CAMPAIGN_TYPE::SCENARIO;
+		return type == campaign_type::type::SCENARIO;
 	}
 
 	bool is_multiplayer() const
 	{
-		return campaign_type == CAMPAIGN_TYPE::MULTIPLAYER;
+		return type == campaign_type::type::MULTIPLAYER;
 	}
 
 	bool is_test() const
 	{
-		return campaign_type == CAMPAIGN_TYPE::TEST;
+		return type == campaign_type::type::TEST;
 	}
 
 	bool is_tutorial() const
 	{
-		return campaign_type == CAMPAIGN_TYPE::TUTORIAL;
+		return type == campaign_type::type::TUTORIAL;
```

Comment:
Side note: this type doesn't even exist anymore, right?

Reply:
It shouldn't be in anything new, but I guess old savegames might have it in.